### DISCUSSION TITLE
Can we use --include-threads all with Docker command?

### DISCUSSION THREAD

**Discussion by eelegiap** [Tyrrrz/DiscordChatExporter#1191] [Q&A] [answered]
I'm hoping to export a channel including all threads. My command looks like below, but I'm hitting this error: `Unrecognized option(s): --include-threads`. I'm wondering why the `--include-threads all` flag is failing here. I am using the Docker image which says it was created 4 days ago. 

```BOT_TOKEN="$1"
CHANNEL_ID="$2"
OUTPUT_NAME="$3"
START="$4"
END="$5"
PATH_TO_RAW_DATA="$6"

docker run --rm \
-v "$PATH_TO_RAW_DATA":/out \
tyrrrz/discordchatexporter:stable export \
-t "$BOT_TOKEN" \
-c "$CHANNEL_ID" \
-f Json \
-o "$OUTPUT_NAME".json \
--after "$START" \
--before "$END" \
--include-threads all```

**Comment by Tyrrrz** [comment, marked as answer]
Hi. The `--include-threads` option is not available on individual channel exports (i.e. the regular `export` command), but only on mass-channel exports (i.e. `exportall`, `exportguild`). Threads are, in essence, just individual channels.

Can you share what kind of result do you want to achieve in your case?

**Reply by eelegiap** [reply]
Hi, thanks for your quick reply and the explanation. I would like to export all channels on a given server.

**Reply by Tyrrrz** [reply]
In that case you should probably use `exportguild` with the command line you had above

**Reply by eelegiap** [reply]
Thanks, will try it out. I appreciate it. This is a great tool!
